django/core/checks/security/base.py

```diff
@@ -107,6 +108,12 @@
     id='security.W021',
 )
 
+W022 = Warning(
+    "Your SECRET_KEY contains an invalid string or UTF-8 bytes. "
```

Comment:
What about this one?
    "Your SECRET_KEY contains an invalid string or UTF-8 bytes. "
    "SECRET_KEY must be a Unicode string in Python3 or a Python 2 "
    "string or UTF-8 bytes.",